src/core-styles/base/_normalize.scss

```diff
@@ -0,0 +1,231 @@
+@import "../mixins/decorations";
+@import "../mixins/spacing";
+@import "../mixins/typography";
+@import "../settings/color";
+@import "../settings/grid";
+@import "../../../node_modules/normalize.css/normalize";
```

Comment:
Can be simplified to:
```
@import "~normalize.css/normalize";
```
Will use node_module resolution, and will ensure it works with Yarn/NPM workspaces. Not massive problem for this repo, but for 3rd-parties including the scss it may be?

Reply:
Is the `~` prefix syntax specific to Storybook?

Reply:
WebPack. (See also https://github.com/libero/reviewer/issues/131#issuecomment-518978629.)